manifests/mod/remoteip.pp

```diff
@@ -0,0 +1,27 @@
+class apache::mod::remoteip (
+  $header            = 'X-Forwarded-For',
+  $proxy_ips         = [ '127.0.0.1' ],
+  $proxies_header    = undef,
+  $trusted_proxy_ips = undef,
+  $apache_version    = $::apache::apache_version
```

Comment:
i'm not sure it's necessary to pass this as parameter, rather just use it as variable, external or localized.